Packs/CrowdStrikeFalcon/Integrations/CrowdStrikeFalcon/README.md

```diff
@@ -6495,21 +6524,69 @@ Retrieve vulnerability details for a specific ID and host. Supported with the Cr
 | --- | --- | --- | --- |  --- | --- |  --- | --- |  --- | --- |
 | CVE-20212-2222 |  host | 1 | Server | ip |  |  | site | 5.5 |  |
 
+
+### cs-falcon-get-events
+
+***
+Get events from CrowdStrike Falcon. This command is intended for development and debugging purposes and should be used with caution as it may create duplicate events. Note that pushing events is supported on Cortex XSIAM only.
+
+#### Base Command
+
+`cs-falcon-get-events`
+
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| fetch_type | Type of incidents or detections. Note that the "On-Demand Scans Detection" option is not available in the legacy version of the API. | Optional |
+| should_push_events | If true, the command will create events, otherwise it will only display them. Possible values are: true, false. Default is false. Note that pushing events is supported on Cortex XSIAM only. | Required |
```

Comment:
```suggestion
| should_push_events | If true, the command will create events, otherwise it will only display them. Possible values are: true, false. Default is false. Note that pushing events is supported in Cortex XSIAM only. | Required |
```